src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -48,6 +50,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Component;
+import org.thymeleaf.util.StringUtils;
```

Comment:
This may have the same behavior, but we don't use this version of `StringUtils` in TDR. Instead we use the one in apache commons.
```suggestion
org.apache.commons.lang3.StringUtils;
```

Although personally I wouldn't use StringUtils at all and instead write `s != null && !s.empty()`.